manifests/plugin/tasks.pp

```diff
@@ -7,19 +7,10 @@
 # $package:: Package name to install, use ruby193-rubygem-foreman-tasks on Foreman 1.8/1.9 on EL
 #            type:String
 #
-# $service:: Service name
-#            type:String
-#
 class foreman::plugin::tasks(
   $package = $foreman::plugin::tasks::params::package,
-  $service = $foreman::plugin::tasks::params::service,
 ) inherits foreman::plugin::tasks::params {
   foreman::plugin { 'tasks':
     package => $package,
-  } ~>
```

Comment:
I still think this is incorrect. It should include the service or else you break 1.14 installations.